workflow/alerting/setup-alert.mdx

```diff
@@ -1,68 +1,69 @@
 
-Select a threshold from the predefined list, you can choose only one threshold at a time. Threshold can help you to define the rule for your alerts.Here's the details of some of the Thresholds.
+### Step 2: Configure Rule
 
--   CPU % :- CPU % is a metric that measures the percentage of time that a CPU (Central Processing Unit) is busy processing data
--   CPU I/O Wait % :- CPU I/O wait % is a metric that measures the percentage of time that a CPU is idle while waiting for input/output (I/O) operations to complete
--   Disk write utilization MB :- Disk write utilization MB is a metric that measures the amount of data written to a disk over a period of time, typically in megabytes (MB).
--   Disk read utilization MB :- Disk read utilization MB is a metric that measures the amount of data read from a disk over a period of time, typically in megabytes (MB).
--   Load average in one minute :- Load average in one minute is a metric that measures the average number of processes that are running or waiting to run on a system over the last one minute. It is a measure of the workload on the system's CPU.
--   Memory used % :- Memory used % is a metric that measures the percentage of the total available physical memory that is currently being used by the system.
+![](https://lh7-us.googleusercontent.com/RlyZPDQYpsVubHR_g3INFrISc9_qe-ulsc41uEWsVPl29tk6D_ISSpT77LeWTiOpeQQH7NZDDiucKWxGhHvm2hBy9mF1fOPYGWlU3PHDwui4sq6J1Rh8jhFwzDjozxkUI2XgIVJQxFtl-xHxBeNaBQo)
 
-### Set alert condition
-You can specify the threshold value at which you wish to notify, as well as other levels such as Warning, and you can also construct custom thresholds.<img src="/images/alert/aa3.png" />
+#### Step 2a: Select Your Data Source
 
-Notify your team
-----------------
-Here you can define the name of the alert, tags with the alert's state, and where you wish to be notified, such as slack or Gmail.\
-To receive alerts on slack, you must have [integrated slack with Middleware](https://docs.middleware.io/docs/integrations/Slack-integration). 
+The Data Source is the foundation upon which your Alert is configured. Create an alert with any OpenTelemetry metric that your team is currently ingesting.
 
-<img src="/images/alert/aa4.png" />
+#### Step 2b: Add Metrics
 
-Default alerts
---------------
-We have created the default alerts for metrics, logs, and K8s so you don't need to create manual alerts and get notified automatically for these alerts. This feature comes with some default rules that you can turn on or off and customize based on your needs. These default alerts will only fire up to 5 times when the condition matching the rule is met.
+Metrics are parts of the Data Source that you are interested in monitoring. Perform mathematical operations on selected metrics and group them by their corresponding attributes.

```

Comment:
This is a great opportunity to link to the Metric Controle Plane page (once we have it). E.g. "For more infromation on metrics collected in your environment, see the Metric Control Plane."